test/omnisharp/omnisharpIntegrationTests/integrationHelpers.ts

```diff
@@ -17,16 +17,6 @@ export interface ActivationResult {
 }
 
 export async function activateCSharpExtension(): Promise<ActivationResult> {
-    const configuration = vscode.workspace.getConfiguration();
-    await configuration.update(
-        'omnisharp.enableLspDriver',
-        process.env.OMNISHARP_DRIVER === 'lsp' ? true : false,
```

Comment:
this env var is never set.  And this configuration update was throwing (and failing tests now that we awaited it).